internal/ingress/controller/config/config.go

```diff
@@ -1001,8 +994,7 @@ func NewDefault() Configuration {
 		DatadogEnvironment:                     "prod",
 		DatadogCollectorPort:                   8126,
 		DatadogOperationNameOverride:           "nginx.handle",
-		DatadogSampleRate:                      1.0,
-		DatadogPrioritySampling:                true,
+		DatadogSampleRate:                      nil,
```

Comment:
This shouldn't be needed, except for consistency

Reply:
It's here for consistency.